### DISCUSSION TITLE
Full Multi-Currency

### DISCUSSION THREAD

**Discussion by brandonkelly** [craftcms/commerce#2280] [Ideas] [answered]
Different primary currency per store?

Full multi-currency prices and system discounts

https://docs.google.com/spreadsheets/d/1amiQvydtxWna6cfEMcSdfeWx7uMY6E-h40oGu5KS6SM/edit

- [x] Use the money/money library #31 
- [ ] Convert all values to int/string

**Comment by adrienne** [comment]
Just tagging myself for updates because i *need* this functionality for a customer i would like to get switched to Craft.

**Comment by adrienne** [comment]
@brandonkelly - any word on this being in the roadmap or anything? I'm very interested!

**Comment by lukeholder** [comment]
Hi @adrienne Check out our roadmap: https://craftcms.com/roadmap

Full multi-currency would fall under our planned Multi-store functionality :)

**Comment by adrienne** [comment]
Oof, all the way til 5.0, huh?

**Comment by jak-kal** [comment]
Tagging for an update because this is a must - otherwise we must purchase and install separate installs for each territory. I don't begrudge the cost but this almost defeats the purpose and is difficult to manage and scale. Luckily we can lift much of the burden by introducing a CRM. Still a no brainer though!

**Comment by wblommaert** [comment]
Can the progress of this feature be tracked anywhere? Would definitely be interesting for one of our projects starting later this year/early next year.

**Comment by lukeholder** [comment, marked as answer]
Craft Commerce 5 was released earlier this year and has full multi-store capability, with a per-store currency for product pricing.
